.github/workflows/dependency-updates.yml

```
@@ -0,0 +1,55 @@
+          cat updates.txt
+          if grep -q "All dependencies match the latest package versions" updates.txt; then
+            echo "No updates available"
+            echo "updates_available=false" >> $GITHUB_OUTPUT
+          else
+            echo "Updates available"
+            echo "updates_available=true" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Create Pull Request
+        if: steps.ncu.outputs.updates_available == 'true'
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.7)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)